projects/core/directives/dropdown/dropdown-hover.directive.ts

```diff
@@ -0,0 +1,36 @@
+import {Directive, Inject, Input} from '@angular/core';
+import {tuiDefaultProp, TuiHoveredService} from '@taiga-ui/cdk';
+import {tuiAsDriver, TuiDriver} from '@taiga-ui/core/abstract';
+import {merge, Observable, of, Subject} from 'rxjs';
+import {delay, switchMap} from 'rxjs/operators';
+
+@Directive({
+    selector: `[tuiDropdownHover]:not(ng-container)`,
+    providers: [tuiAsDriver(TuiDropdownHoverDirective), TuiHoveredService],
+})
+export class TuiDropdownHoverDirective extends TuiDriver {
+    private readonly toggle$ = new Subject<boolean>();
+    private readonly stream$ = merge(this.toggle$, this.hovered$).pipe(
+        switchMap(visible =>
+            of(visible).pipe(delay(visible ? this.showDelay : this.hideDelay)),
+        ),
+    );
+
+    @Input(`tuiDropdownShowDelay`)
+    @tuiDefaultProp()
+    showDelay = 200;
```

Comment:
what do you think about?

```ts
    @Input(`tuiDropdownShowDelay`)
    @tuiDefaultProp()
    showDelay = this.options.showDelay;

    @Input(`tuiDropdownHideDelay`)
    @tuiDefaultProp()
    hideDelay = this.options.hideDelay;
```
```

Reply:
I don't want to introduce show/hide delay to options because they will only be used in one of the drivers. Maybe later if there's a need.